src/app.js

```diff
@@ -3,11 +3,12 @@ import axios from 'axios';
 const countriesDiv = document.getElementById('countries');
 
 
```

Comment:
const randomCountries = ['Peru', 'Italy', 'Germany', 'Poland', 'Finland', 'China', 'Indonesia', 'Argentina', 'Brazil',
'Belarus', 'Turkey', 'Netherlands', 'France', 'Spain', 'Portugal', 'Switzerland', 'Greece', 'Norway','Ireland',
'Hungary', 'Thailand', 'Vietnam', 'Mongolia', 'Australia', 'Ghana', 'Canada', 'Mexico'];

probeer het ook eens met deze array.

Reply:
**Als je in de Main branch kijkt zie je dat ik hiervoor een andere code had gemaakt!
Toen ik later zag dat de landen gesorteerd op populatie moesten worden weergegeven heb ik deze code in elkaar gezet!
Voor vragen voel je vrij me een chat te sturen in teams!**